src/checkstyle/checkstyle-header.txt

```diff
@@ -13,5 +13,4 @@
 ^\Q * See the License for the specific language governing permissions and\E$
 ^\Q * limitations under the License.\E$
 ^\Q */\E$
-^$
```

Comment:
OK. I really like that extra line after Copyright.
Let me know if I should push the fix for all guilty classes.